prediction_market_agent/agents/metaculus_agent/deploy.py

```diff
@@ -62,22 +60,27 @@ def verify_market(self, market_type: MarketType, market: AgentMarket) -> bool:
         # Otherwise all markets on Metaculus are fine.
         return True
 
+    def build_random_probabilistic_answer(self) -> ProbabilisticAnswer:
+        return ProbabilisticAnswer(
+            p_yes=Probability(0.5),
```

Comment:
That's fixed, not random.

Reply:
Yep, renamed to fixed - that's also the previous behavior if I understand it correctly